lambdatimes/src/components/Content/Cards.js

```diff
@@ -1,15 +1,31 @@
 import React, { Component } from 'react';
 import Card from './Card';
+import PropTypes from 'prop-types';
 
 const Cards = props => {
+  console.log(props)
   return (
     <div className="cards-container">
       {/* Using the cards prop, map over the list creating a 
           new Card component for each passing the card as the only prop*/}
+      {props.cards.map((card,index)=>
+      
+      <Card 
+        card={card}
+        key={index}
+      />)}
+
     </div>
   )
 }
 
 // Make sure you include prop types for all of your incoming props
+Cards.propTypes= PropTypes.shape({
```

Comment:
Note that propTypes should be an object here, with the key being the name of the prop and the value being the PropTypes.shape()